examples/03_evaluate/README.md

```diff
@@ -20,6 +21,11 @@ Two approaches for evaluating model performance are demonstrated along with thei
 3. Classification metrics: These are used to evaluate binary labels
     * Arear Under Curver (AUC) - integral area under the receiver operating characteristic curve
     * Logistic loss (Logloss) - the negative log-likelihood of the true labels given the predictions of a classifier
+4. Non accuracy based metrics: These do not require any ground truth but instead evaluate the following properties of the recommendations
```

Comment:
doesn't serendipity still use ground truth, i guess it's optional but alternatively you could just say:

`4. Non accuracy based metrics: These do not compare predictions against ground truth but instead evaluate the following properties of the recommendations`

Reply:
Good point. I made this change.